src/Kernel/Constants/Rights.cs

```diff
@@ -24,9 +24,9 @@ public static class Rights
     public const int AddEditRemoveNews = 5;
     public const int AddEditRemovePositions = 6;
     public const int AddEditRemoveTime = 7;
-    public const int AddEditRemoveHistories = 8; // can be reused
+    public const int AddEditRemoveOffices = 8;
     public const int AddEditRemoveCompanies = 9;
-    public const int AddRemoveDepartmentData = 10; // reuse the int, we do not need the right 
+    public const int AddEditRemoveBooking = 10;
```

Comment:
i think we need to update these rights' localisations in rights service's db

Reply:
great remind